src/content/smart-bulletins/033.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Smart Bulletin No. 033"
+intro: "Card Management Implications and Information for Agency Consideration in Relation to the Section 889 Prohibition on Acquiring Certain Telecommunications and Video Surveillance Equipment"
+number: "No. 033"
+order: 033
+businessline: "Purchase, Travel, Fleet and Integrated"
+published: "Aug 20, 2020"
+lastupdated: "N/A"
+---
+
+UPDATE:
+| Date | Version History Action Log - Summary of Action/Changes |
+| ----------- | ----------- |
+| August 20, 2020 | Implementation of Smart Bulletin #33 |
+
+## Effective Date
+
+August 20, 2020
+
+## Business Line(s) Affected
+
+Purchase, Travel, Fleet and Integrated
+
+## Introduction
+
+This Smart Bulletin, which augments Smart Bulletin 029 effective August 13, 2019, provides guidance to agencies implementing section 889(a)(1)(B) (“Part B”) of the John S. McCain National Defense Authorization Act (NDAA) for Fiscal Year (FY) 2019 (Pub. L. 115-232), which prohibits Federal agencies from purchasing products or services from entities that use covered equipment or services as a substantial or essential component of any system, or as critical technology as part of any system, regardless of whether that usage is in performance of work under Federal contract. This prohibition became effective August 13, 2020. This prohibition applies to all contracts, regardless of dollar amount, including micro-purchases on charge cards as well as formal contracting actions. (The full text of the related Federal Acquisition Regulation (FAR) interim rule on section 889 can be viewed at: https://www.regulations.gov/document?D=FAR-2019-0009-0003.)
```

Comment:
This Smart Bulletin, which augments Smart Bulletin 029 effective August 13, 2019, provides guidance to agencies implementing section 889(a)(1)(B) (“Part B”) of the John S. McCain National Defense Authorization Act (NDAA) for Fiscal Year (FY) 2019 (Pub. L. 115-232), which prohibits Federal agencies from purchasing products or services from entities that use covered equipment or services as a substantial or essential component of any system, or as critical technology as part of any system, regardless of whether that usage is in performance of work under Federal contract. This prohibition became effective on August 13, 2020. This prohibition applies to all contracts, regardless of dollar amount, including micro-purchases on charge cards as well as formal contracting actions. (The full text of the related Federal Acquisition Regulation (FAR) interim rule on section 889 can be viewed at: https://www.regulations.gov/document?D=FAR-2019-0009-0003.)
The “Part A” prohibition (Sec 889(a)(1)(A)), prohibiting the purchase of equipment/services containing covered technology, went into effect on August 13, 2019, and is discussed in Smart Bulletin 029. As of the date of this Smart Bulletin, “covered equipment/services” include those provided or produced by the following entities and their subsidiaries and affiliates:
Huawei Technologies Company
ZTE Corporation
Hytera Communications Corp
Hangzhou Digital Hikvision Technology Company
Dahua Technology Company
Agencies are required to develop policies regarding their use of charge cards in the context of both Section 889 prohibitions (Parts A and B). This Smart Bulletin provides operational guidance and information on issues agencies participating in the GSA SmartPay program may desire to consider as they develop those policies, including section 889 implications for all business lines (purchase, travel, and fleet cards).
This Smart Bulletin uses summary language to address complex issues that may develop under certain purchasing circumstances. When such circumstances exist or are anticipated to arise related to a planned purchase, cardholders and card managers are encouraged to consult with agency acquisition, information technology, and/or supply chain risk management policy officials in accordance with agency policy.
This Smart Bulletin does not address section 889-related requirements related to formal contracting procedures. Instead, it focuses on section 889 implications with regard to purchase card micro-purchases, travel, and fleet card transactions. Because integrated cards combine these account types, they are not separately addressed.